changelog.json

```
@@ -1,5 +1,9 @@
 {
   "releases": {
+    "3.0.1-beta2": [
+      "[Fixed] Stream output from merge-tree - #14485",
```

Not sure if this is the best wording (any suggestions @niik?), but definitely related to what the users experienced.
```suggestion
      "[Fixed] Fix crash determining mergeability of a branch with many conflicts - #14485",
```

---

Yeah, that's a great improvement. How about something like

```suggestion
      "[Fixed] Prevent crash when encountering a large number of conflicts while checking for ability to merge automatically - #14485",
```

A bit wordy but uses the terminology of desktop and .com rather than my (possibly) made up word "mergeability"

---

Sold! I'd say we can deploy this one already